src/LibraryManager/Providers/FileSystem/FileSystemCatalog.cs

```diff
@@ -136,6 +136,25 @@ public Task<CompletionSet> GetLibraryCompletionSetAsync(string value, int caretP
             return library;
         }
 
+        public string GetSuggestedDestination(ILibrary library)
+        {
+            if (library != null && library is FileSystemLibrary fileSystemLibrary)
```

Comment:
Do we need anything for the file system? I guess you can just return empty string.